### PR TITLE
Workaround for more sources

### PR DESCRIPTION
**Pull request by githubus11**
Add time.sleep because of possible 503
Ignore "deleted" sources

Drawback: Getting sources now takes at least 3 seconds, one per Server (OpenLoad, F1, F2)

### CODE REVIEW

**Comment by githubus11** [comment]
maybe we should filter OpenLoad from get_episode_sources() as it wont be parsed in the near future anywhere. As we only have a limited amount of serverrequest per time without getting a 503 it would save us one request in embed_extractor.load_video_from_url where we try to get the pageContent openLoad

File: embed_extractor.py [hunk 1]
```diff
@@ -14,6 +15,7 @@ def load_video_from_url(in_url):
         reqObj = http.send_request(in_url)
         page_content = reqObj.text
         url = reqObj.url
+        time.sleep(1)
```
Comment (DxCx) [inline embed_extractor.py]: Thanks once more :)
Please just move sleep into 9anime_direct
This will ensure we wont sleep on openload 👍

**Comment by githubus11** [comment]
done
You should integrate a loggin facility so we can log all errors we get and show the user a meaningful message instead of throwing and catching exceptions

**Comment by DxCx** [comment]
Thanks!

**Pull request merged**